### DISCUSSION TITLE
Custom split direction?

### DISCUSSION THREAD

**Discussion by vunhatchuong** [hyprwm/Hyprland#7064] [Q&A]
No matter what layout I use, the 3rd app will always open next to the 2nd one, I want to change so that the 3rd one is below the 2nd one instead. Kinda like MonadTall.

**Comment by ravxen** [comment]
With the "dwindle" layout and default keybinds, you can press "SUPER + J" to toggle the splitting mode.

See: https://wiki.hyprland.org/Configuring/Dwindle-Layout/ -> Layout Messages
